internal/service/message_handler_test.go

```
@@ -1142,6 +1143,218 @@ func TestMessageHandlerOrchestrator_UsernameToSub_NoUserReader(t *testing.T) {
+		inputUser := &model.User{
+			Token:    "test-token",
+			Username: "testuser",
+			UserID:   "auth0|testuser",
+			UserMetadata: &model.UserMetadata{
+				Name: converters.StringPtr("Test"),
+			},
+		}
+		data, _ := json.Marshal(inputUser)
+		msg := &mockTransportMessenger{data: data}
+
+		orchestrator.UpdateUser(ctx, msg)
```

golangci-lint (2.11.4)

[error] Error return value of `orchestrator.UpdateUser` is not checked

(errcheck)